exercises/pangram/PangramExample.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
+// Needed to strip diacritics
+
+struct Pangram {
+
+    static func isPangram(_ text: String) -> Bool {
```

Comment:
Ah interesting, I think it is cleaner to extend builtin types. This is a simple exercise so I agree on putting on struct but it would be cool to figure out a way to require the test to use protocols. 

```swift
protocol Pangram {
   var isPangram:Bool
}